hub/package-meta.yaml

```diff
@@ -1,7 +1,7 @@
 ---
```

Comment:
we can delete this file. This is autogenerated:

Reply:
done